### DISCUSSION TITLE
[Announment] Welcome new PMC Member: Xiaoquan Wang

### DISCUSSION THREAD

**Discussion by nobodyiam** [apolloconfig/apollo#4144] [Announcements]
Hi Community,

On behalf of the Apollo PMC, I am pleased to announce that Xiaoquan Wang a.k.a Anilople, has been voted in as a new Apollo PMC Member.

Please join me to say congratulations to him!

@Anilople, would you please briefly introduce yourself to the community?

Best Regards,
Jason

**Comment by zouyx** [comment]
Congratulation to @Anilople and welcome!

**Reply by Anilople** [reply]
Thanks zou gege @zouyx

**Comment by Anilople** [comment]
Thanks everyone. I am very happy to join Apollo PMC.
